source/sensdet/SensorHit.h

```diff
@@ -40,9 +40,9 @@ namespace nexus {
      void operator delete(void*);
 
     /// Returns the detector ID code
-    G4int GetPmtID() const;
+    G4int GetSensorID() const;
     /// Sets the detector ID code
-    void  SetPmtID(G4int);
+    void  SetSensorID(G4int);
```

Comment:
The code is now consistent, although it may be that some users must modify their own custom classes. Do you think that it's better to separate these changes in a different PR?

Reply:
I am not familiar with how frequently this is used. If (in practice) nobody will notice, then it's probably ok.

Reply:
It's not very frequent, it would be the case of a user writing a different persistency manager, which I think that does not happen a lot.